simulator/src/main/java/com/hedera/block/simulator/BlockStreamSimulatorApp.java

```diff
@@ -148,8 +134,10 @@ public void stop() throws InterruptedException {
     public StreamStatus getStreamStatus() {
         return StreamStatus.builder()
                 .publishedBlocks(publishStreamGrpcClient.getPublishedBlocks())
+                .processedBlocks(publishStreamGrpcServer.getProcessedBlocks())
```

Comment:
If all grpc implementations are unifed under one interface, this could be improved and reduced.